tests/integration/helpers.py

```diff
@@ -139,6 +139,29 @@ def restart_unit(model_full_name: str, unit: str) -> None:
     )
 
 
+def access_prometheus_exporter(host: str) -> bool:
+    """Check if a given unit has 'kibana-exporter' service available and publishing."""
+    try:
+        # Normal IP address
+        socket.inet_aton(host)
+    except OSError:
+        socket.inet_pton(socket.AF_INET6, host)
+        host = f"[{host}]"
+
+    url = f"http://{host}:9684/metrics"
+    response = requests.get(url)
```

Comment:
We need to handle http exceptions here - and return False if it fails

Reply:
Done.